sdk/src/client/error.rs

```diff
@@ -217,6 +201,13 @@ pub enum Error {
     Stronghold(#[from] crate::client::stronghold::Error),
 }
 
+// Used in faucet funds request util
+impl From<reqwest::Error> for Error {
```

Comment:
If this Error can be created from a reqwest error, then the variant should probably just stay.

Reply:
I felt like since its only the faucet, which is sort of a "node" its better to not make the From explicit for entire crate. Would it be better if i made the error from the funds request directly?

Reply:
You mean create a `Faucet` variant in the error?

Reply:
Hmm didnt think of that, maybe thats a good idea!